ci/gpu/build.sh

```diff
@@ -98,9 +98,13 @@ fi
 # TEST - Run GoogleTest and py.tests for libcugraph and cuGraph
 ################################################################################
 
-set +e -Eo pipefail
```

Comment:
Not sure if this was intentional, but I don't see the `set -E` carried over in your changes.

Reply:
added in next commit along with the trap.